cmd/juju/machine/add.go

```diff
@@ -64,7 +65,7 @@ Examples:
    juju add-machine lxd -n 2             (starts 2 new machines with an lxd container)
    juju add-machine lxd:4                (starts a new lxd container on machine 4)
    juju add-machine --constraints mem=8G (starts a machine with at least 8GB RAM)
-   juju add-machine ssh:user@10.10.0.3   (manually provisions a machine with ssh)
+   juju add-machine ssh:user@10.10.0.3   (manually provisions a linux machine with ssh)
```

Comment:
We may end up adding support for other *nixes, so I think this should be left as it was.